### PR TITLE
feat(old age pension): New application

### PR DESCRIPTION
**Pull request by qdd1**
# ...

Attach a link to issue if relevant

## What

New application: old age pension
It is still in development. We merge it to main to test connections and other things.
NOT GOING OUT TO PROD

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/service-portal/information/src/lib/messages.ts [hunk 1]
```diff
@@ -51,7 +51,7 @@ export const spmm = defineMessages({
     defaultMessage:
       'Íslendingur með lögheimili á Íslandi (t.d. námsmaður eða sendiráðsmaður); í hjúskap með útlendingi sem ekki er á skrá',
   },
-  maritalStatusMarried: {
+  maritalþStatusMarried: {
```
Comment (thorkellmani) [inline libs/service-portal/information/src/lib/messages.ts:54]:
```suggestion
  maritalStatusMarried: {
```
I'm guessing this was unintended

**Review by thorkellmani — CHANGES_REQUESTED** [review]
This is probably an accident

File: libs/clients/national-registry/v2/src/lib/nationalRegistryClient.service.ts [hunk 2]
```diff
@@ -108,6 +108,13 @@ export class NationalRegistryClientService {
     return formatCohabitationDto(response)
   }
 
+  async getCohabitants(nationalId: string): Promise<string[] | null> {
+    const cohabitants = await this.handleLegacyMissingData(
+      this.individualApi.einstaklingarGetSamibuarRaw({ id: nationalId }),
+    )
+    return cohabitants || null
```
Comment (eirikurn) [inline libs/clients/national-registry/v2/src/lib/nationalRegistryClient.service.ts:115]: Should default to [] for consistency?

**Review by eirikurn — COMMENTED** [review]
Reviewed core files.

**Review by eirikurn — APPROVED** [review]
Approving core files

**Review by Thorrik58 — CHANGES_REQUESTED** [review]
Feature flag is needed and there is an inconsistency with the fields, there are also a lot of comments that havent been removed too but I didnt want to mention them all.

**Pull request closed without merging**